tests/utils.go

```diff
@@ -972,7 +970,7 @@ func CreateSecret(name string, data map[string]string) {
 }
 
 func CreateHostPathPVC(os, size string) {
-	CreatePVC(os, size, Config.StorageClassHostPath, false)
+	CreatePVC(os, size, Config.StorageClassLocal, false)
```

Comment:
I don't think this hsould be StorageClassLocal, shouldn't it be the storage class you defined in the test?